pysgrid/processing_2d.py

```diff
@@ -22,8 +23,7 @@ def vector_sum(x_arr, y_arr):
     :rtype: numpy.array
 
     """
-    vector_sum = np.sqrt(x_arr**2 + y_arr**2)
-    return vector_sum
+    return ma.sqrt(x_arr**2 + y_arr**2)
```

Comment:
User are scared when the see the invalid values warning. We could be smarter and return a numpy array when the input is a numpy array and return a masked array when the input is a masked a array (I have a decorator for that). However, one could argue that the result of a velocity vector should always be a masked array as invalid values will be virtually always present. (And the decorator would make this simple function too complex and slow.)